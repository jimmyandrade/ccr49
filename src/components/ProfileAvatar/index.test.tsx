import {create} from "react-test-renderer";
import {ProfileAvatar} from "./index";

jest.mock("next/image", () => "Image");

describe("<ProfileAvatar /> component", () => {
  it("should match snapshot", () => {
    const testRenderer = create(<ProfileAvatar alt={"Test Alt Text"} />);
    expect(testRenderer).toMatchSnapshot();
  });
});
