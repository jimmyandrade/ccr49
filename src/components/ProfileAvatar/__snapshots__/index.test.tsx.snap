// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProfileAvatar /> component should match snapshot 1`] = `
<div
  className="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault"
>
  <Image
    alt="Test Alt Text"
    height={40}
    src="test-file-stub"
    width={40}
  />
</div>
`;
